tests/tests/project.rs

```diff
@@ -106,12 +106,12 @@ impl ProjectBuilder {
             r#"
                 [[bin]]
                 name = "{name}"
-                path = "{path}"
+                path = {path}
                 test = false
                 doc = false
             "#,
             name = name,
-            path = path.display(),
+            path = toml::to_string(&path).unwrap(),
```

Comment:
Is this change unrelated? If so, could you split it out into its own PR. I'm also not sure what the motivation for this change is.

Reply:
It is required for Windows support because of Windows paths.